File: Taller2/C#/Taller2_CS/Taller2_CS/Model/DBConnection.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Npgsql;

namespace Taller2_CS
{
    class DBConnection
    {
        string connection = "Server=Localhost;Port=5432;Database=taller2;User Id=postgres;Password=admin";
        NpgsqlConnection conn;

        public NpgsqlConnection Connection()
        {
            conn = new NpgsqlConnection(connection);
            conn.Open();

            return conn;
        }
    }
}

File: Final/API/model/db_connection.js
var pgp = require('pg-promise')(),
    conf = require('./db_config'),
    conn;
const connectionData = {
    user : conf.user,
    host:conf.host,
    database:conf.database,
    password:conf.password,
    port:conf.port
}

conn = pgp(connectionData);


module.exports = conn;
